understanding/22/target-size-minimum.html

```diff
@@ -77,13 +77,21 @@ <h2>Intent of Target Size (Minimum)</h2>
             <figcaption>A dropdown menu is shown. The PASS example has menu items which are 24 px high. In the FAIL example, the menu items are only 20 px high.</figcaption>
         </figure>
 
-        <p>Where targets are embedded inside other targets, the inner target either needs a size of at least 44 x 44 CSS pixels or must be placed at the edge or corner to make sure that there is sufficient adjacent spacing in each dimension.</p>
-
         <figure id="pointer=target-embedded">
-            <img src="img/pointer-target-example5.png" alt="Three examples of a target adjacent to, or nested within another target (see legend below)" />
-          <figcaption>Three examples of a person's profile image, which is supposed to be a target leading to another page, and a magnifying glass icon, which is supposed to open a lightbox view of the profile image. In the first example, the magnifying icon is outside the picture with sufficient spacing and passes. In the second example, the icon overlaps with the top-left corner of the image, but passes due to sufficient spacing top and left. In the third example, the icon sits inside the linked image. Since it is surrounded by an active click area, the icon target is not large enough to pass the Success Criterion.</figcaption>
+            <img src="img/pointer-target-example5-revised.png" alt="Three examples of a target adjacent to, or nested within another target (see legend below)" />
+          <figcaption>Three examples of a person's profile image, which is supposed to be a target leading to another page, and a magnifying glass icon, which is supposed to open a lightbox view of the profile image. In the first example, the magnifying icon is outside the picture Its horizontal dimension is 24px. The vertical dimension is below 24px but since there are no adjacent targets above and belowm, the target meets the Success Criterion. In the second example, the icon overlaps with the top-left corner of the image, The image, itself a target, is immediately adjacent to the loupe icon. Measuring from the top of the loupe icon to the nearest point of the image target shows that that the offset is only 20px. The second example therefore fails the Success Criterion. In the third example, the loupe icon sits inside the linked image. Since it is surrounded by a larger target, the icon target fails - the offset to the surrounding and immediately adjacent image target is only 20px at top and bottom.</figcaption>
```

Comment:
"outside of the picture Its horizontal dimension is 24px" missing a full stop "outside of the picture. Its horizontal dimension is 24px"

Reply:
typo: "belowm" > "below"

Reply:
"left corner of the image, The image, itself a target" change comma to full stop "left corner of the image. The image, itself a target"